programs/programs.json

```diff
@@ -350,5 +350,17 @@
     "categories": [
       "NFT"
     ]
+  },
+    {
+    "name": "soundwork-contracts",
```

Comment:
Nit: this file is currently ordered alphabetically by the `name` property, so it would be great if you could move this below [the `solend` entry](https://github.com/solana-playground/assets/blob/795daaef9d269379de8c066bad5c2f5ba6253551/programs/programs.json#L116-L126).